api-doc/openapi.yaml

```diff
@@ -380,35 +382,56 @@ components:
         amount:
           type: number
       required:
-      - timestamp
-      - message
-      - amount
+        - id
+        - timestamp
+        - message
+        - amount
       example:
+        id: 6c28df04-b9d2-4ebe-bd33-58c9b290d929
         timestamp: '2021-09-21T15:05:47Z'
         name: jsloth
         message: ota tää :D 🤔🤔
         amount: 10
-    Donations:
+    DonationWithoutId:
       type: object
       properties:
-        id:
-          $ref: '#/components/schemas/Donation'
+        timestamp:
+          type: string
```

Comment:
Use date-time here, and on other timestamps as well. See https://swagger.io/docs/specification/data-models/data-types/